app-backend/database/src/main/scala/com/azavea/rf/database/fields/BandDataTypeField.scala

```diff
@@ -0,0 +1,8 @@
+package com.azavea.rf.database.fields
+
+import com.azavea.rf.database.ExtendedPostgresDriver.api._
+import com.azavea.rf.datamodel.BandDataType
+
+trait BandDataType { self: Table[_] =>
```

Comment:
whoops, I don't think I event need the field type since it's just a part of the jsonb. Case classes should be handling the actual validation.